resources/web/streaming_viewer/index.html

```diff
@@ -23,7 +23,7 @@ <h1>Webots streaming viewer</h1>
       </p>
     </header>
     <webots-view style="height:calc(100% - 120px);"></webots-view>
-    <script type="module" src="https://cyberbotics.com/wwi/R2022b/WebotsView.js"></script>
+    <script type="module" src="../wwi/WebotsView.js"></script>
```

Comment:
Shouldn't we keep an absolute reference here instead of a relative one?

Reply:
yes, I forgot to change it back